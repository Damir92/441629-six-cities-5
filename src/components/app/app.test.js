import React from 'react';
import {shallow} from 'enzyme';

import App from './app';

describe(`App renders correctly`, () => {
  it(`App renders correctly`, () => {
    const tree = shallow(
        <App />
    );

    expect(tree).toMatchSnapshot();
  });
});
