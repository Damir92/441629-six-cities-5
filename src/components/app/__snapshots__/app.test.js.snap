// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly App renders correctly 1`] = `
<Router
  history={
    Object {
      "action": "POP",
      "block": [Function],
      "createHref": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "length": 1,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "pathname": "/",
        "search": "",
        "state": undefined,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(MainPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(WithAuthForm)",
          "type": [Function],
        }
      }
      exact={true}
      path="/login"
    />
    <Connect(PrivateRoute)
      path="/favorites"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/favorites"
    >
      <FavoritesPage />
    </Route>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(OfferPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/offer/:id"
    />
  </Switch>
</Router>
`;
